ui/app/components/page-container/page-container-footer/page-container-footer.component.js

```diff
@@ -25,19 +26,20 @@ export default class PageContainerFooter extends Component {
       submitText,
       disabled,
       submitButtonType,
+      hideCancel,
     } = this.props
 
     return (
       <div className="page-container__footer">
 
-        <Button
+        {!hideCancel && <Button
```

Comment:
Would it be enough to check for an `onCancel` function prop here? Or would you prefer to explicitly have a `hideCancel` prop?

Reply:
I mostly just did this as extra assurance that my change would be backwards compatible. I guess I am partial to explicitly using a `hideCancel` prop; the separation between code responsible for presentation and code responsible for handling user action feels cleaner that way.